lib/s3storage.php

```diff
@@ -312,7 +312,8 @@ public function getDirectDownload($urn, $versionId = null) {
 		$cmd = $this->connection->getCommand('GetObject', [
 			'Bucket'    => $this->getBucket(),
 			'Key'       => $urn,
-            'VersionId' => $versionId
+			'VersionId' => $versionId,
+			'ResponseContentType' => 'application/octet-stream'
```

Comment:
maybe also add 'ResponseContentDisposition' => 'attachment; filename=<filename>',  ... problem is we don't have the filename here